config/src/config/mempool_config.rs

```diff
@@ -43,6 +44,7 @@ impl Default for MempoolConfig {
             system_transaction_timeout_secs: 600,
             system_transaction_gc_interval_ms: 60_000,
             shared_mempool_validator_broadcast: true,
+            shared_mempool_early_expiry_secs: 20,
```

Comment:
So the problem here is I can't then submit a quick submit transaction right?

If let's say I'm trying to do something that must happen in 20 seconds or I don't want it to go through e.g. a price changes significantly on a trade or something similar.  Basically just making it so that you have a minimum expiry time right?

If you were to add this, I would rename it to minimum_expiry_secs